docs/Makefile

```diff
@@ -87,9 +87,9 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/MXCuBE3.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/MXCuBEWeb.qhcp"
 	@echo "To view the help file:"
-	@echo "# assistant -collectionFile $(BUILDDIR)/qthelp/MXCuBE3.qhc"
+	@echo "# assistant -collectionFile $(BUILDDIR)/qthelp/MXCuBEWeb.qhc"
```

Comment:
_[Side note: we should probably clean up the whole Makefile, it is full of things we do not use or need. For example this change here, I am not sure it would still work, no idea because I have never ever used this part of Sphinx Makefile. Maybe I will take the time to clean this up next time I work on the Sphinx things, I probably should have done it on the first round of Sphinx doc reboot.]_

Reply:
I will put it on my (mental) todo list :)